addon/utils/game.py

```diff
@@ -1,19 +1,38 @@
 from pathlib import Path
 from ..utils.common import resolve
-
+import os
 
 def update_game(self, context):
-    self['game'] = resolve(self.game)
+    g = resolve(self.game)
+    # Don't bother re-resolving paths if we haven't changed our game path
+    if g == self['game']:
+        return
+
+    self['game'] = g
```

Comment:
As far as I know, Blender does not call an update function if the value did not change, so this check shouldn't be necessary.